crux-agent/examples/env_context_config_example.py

```
@@ -0,0 +1,166 @@
+    
+    print("=== Current Context Configuration ===")
+    print(f"Context Limit: {settings.lmstudio_context_limit:,} tokens")
+    print(f"Response Reserve: {settings.lmstudio_response_reserve:,} tokens")
+    print(f"Summarization Threshold: {settings.lmstudio_summarization_threshold:.1%}")
+    print(f"Available for Context: {settings.lmstudio_context_limit - settings.lmstudio_response_reserve:,} tokens")
+    
+    # Show what this means in practice
+    available = settings.lmstudio_context_limit - settings.lmstudio_response_reserve
+    threshold_usage = int(available * settings.lmstudio_summarization_threshold)
+    
+    print(f"\nContext Management Behavior:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)